localdev/configs/ocw-course-site-config.yml

```diff
@@ -25,6 +25,23 @@ collections:
           - { label: "City", name: "city", widget: "string", required: true }
           - { label: "State", name: "state", widget: "string", required: true }
           - { label: "Zip Code", name: "zip", widget: "string", required: true }
+      - {
+        "label": "Authors",
+        "name": "authors",
+        "widget": "relation",
+        "collection": "author",
+        "display_field": "title",
+        "multiple": true,
+        "max": 10,
+        "min": 1,
+        }
+
+  - name: author
+    label: Author
+    category: Content
+    folder: content
```

Comment:
This might point to the need for a different type of data collection. As this is written, each author you enter will create a `WebsiteContent` record. Right now our concept of `WebsiteContent` is that it represents user-entered data that will be used to create a file for the SSG, whether that's markdown, json, etc. As this is written, each individual author you enter will create a markdown file in the `content` folder in Github. That's definitely not something we want. All we want to do with this content type is to fill in the valid entries that can be used by a different field. I don't have any great ideas off the top of my head to implement that, but it seems like we'd at least want another model, because the current concept of `WebsiteContent` doesn't really fit here. It's probably worth discussing w/ Ferdi and the rest of the team

Reply:
Sure, we could change this to be a video or something like that where we'd want to have it have it's own page. The idea is just to have the relation field capture relation between two pieces of content. My understanding was that this `ocw-course-site-config.yml` was basically like a placeholder anyhow - for the purposes of development is there something I should be editing instead?

Reply:
OK I'm on the same page now. I think it would be helpful to follow the example in your RFC more closely and add a "Profession" field or something like that to the "author" content type. Your example is fine to exercise the widget, but the intent is a little confusing with just this single field. We can worry about moving non-OCW-course-site items out of this config in a separate issue. I think it's fine to leave it here